src/app/shell/admin-tools/data/history-log/history-log-filters/history-log-filters.component.html

```diff
@@ -5,20 +5,19 @@
       <mat-form-field appPlaceholderStyling class="date" appearance="fill">
         <mat-label>{{ 'FORMS.PLACEHOLDERS.ENTER_DATE_RANGE' | translate }}</mat-label>
         <mat-date-range-input [rangePicker]="picker" [max]="maxDate">
-          <input matStartDate placeholder="{{ 'FROM' | translate }}" formControlName="dateFrom" [tabIndex]="notAllowedToPickByTabButton" />
-          <input matEndDate placeholder="{{ 'TO' | translate }}" formControlName="dateTo" [tabIndex]="notAllowedToPickByTabButton" />
+          <input matStartDate placeholder="{{ 'FROM' | translate }}" formControlName="dateFrom"
+            [tabIndex]="notAllowedToPickByTabButton" />
+          <input matEndDate placeholder="{{ 'TO' | translate }}" formControlName="dateTo"
+            [tabIndex]="notAllowedToPickByTabButton" />
         </mat-date-range-input>
         <mat-datepicker-toggle matSuffix [for]="picker"></mat-datepicker-toggle>
         <mat-date-range-picker #picker (closed)="setDateForFilters()"></mat-date-range-picker>
       </mat-form-field>
     </div>
     <ng-container *ngFor="let filter of filtersList">
       <mat-form-field floatLabel="never" appearance="none">
-        <mat-select
-          disableOptionCentering
-          [formControl]="filtersForm.controls[filter.controlName]"
-          panelClass="dropdown-panel"
-          class="select">
+        <mat-select disableOptionCentering [formControl]="filtersForm.controls[filter.controlName]"
```

Comment:
We probably need to figure out which approach is better for us.
Because in each PR we use different formatting when there are many attributes/inputs

Reply:
I resolve this conversation for now. 
We may have to create a best practices doc, I'll look at it later.